src/pydata_sphinx_theme/assets/styles/sections/_sidebar-secondary.scss

```diff
@@ -24,13 +24,25 @@
   }
 }
 
-.bd-toc {
-  -ms-flex-order: 2;
-  order: 2;
-  max-height: calc(100vh - 5rem);
+.bd-sidebar-secondary {
+  background-color: var(--pst-color-background);
+  padding-top: 1em;
   overflow-y: auto;
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  z-index: var(--pst-level-2);
+  min-height: 100vh;
+  border-left: 1px solid var(--pst-color-border);
   top: var(--pst-header-height);
```

Comment:
On narrow widths I think we should just make the sidebar drawers take up the full vertical space and overlap the header. I don't think they need to be underneath the header. This will create more vertical space for them, and people can easily close by clicking outside of the sidebar anyway. I think this should also make it easier to get the behavior we want

Reply:
If I overlap the header, I wont see the label icon anymore. I think we need to keep some overlay available to exit the sidebar. maybe extend it to 8 colums ?

Reply:
Isn't the label icon already hidden when the sidebars are open on mobile anyway? I am *just* talking about the mobile view, not the wide view. My assumption is that exiting the sidebar is accomplished by clicking outside of the open sidebar on mobile, right?

Reply:
ok I don't know why I read horizontal space. It's done in the latest version: `height: 100vh`